backend-rust/src/indexer.rs

```diff
@@ -4061,6 +4114,11 @@ impl PreparedPayDayBlock {
         // Save the `rewards` to the database.
         self.mint_reward.save(tx, None).await?;
 
+        // Save the commission rates to the database.
+        for event in &self.payday_commission_rates {
+            event.save(tx).await?;
+        }
```

Comment:
We probably need to batch these into one query, otherwise the processing goes up due to the 'chatting'.